src/views/LoanDetails/LoanDetails.js

```diff
@@ -406,14 +406,16 @@ class LoanDetails extends React.Component {
                     <FormattedMessage id="stripes-smart-components.cddd.changeDueDate" />
                   </Button>
                 </IfPermission>
-                <Button
-                  data-test-declare-lost-button
-                  disabled={buttonDisabled || isDeclaredLostItem}
-                  buttonStyle="primary"
-                  onClick={() => declareLost(loan)}
-                >
-                  <FormattedMessage id="ui-users.loans.declareLost" />
-                </Button>
+                <IfPermission perm="ui-users.loans.declare-item-lost">
+                  <Button
+                    data-test-declare-lost-button
+                    disabled={buttonDisabled || isDeclaredLostItem}
+                    buttonStyle="primary"
+                    onClick={() => declareLost(loan)}
+                  >
+                    <FormattedMessage id="ui-users.loans.declareLost" />
+                  </Button>
+                </IfPermission>
               </span>
             </Row>
```

Comment:
@SergiyVSergiyenko, I am wondering if user does not have any of those permissions, will current row display as empty one? Should we check  if user has any permission then render `Row` etc ?

Reply:
Good point, though in this case I'm not worried about it. The `<Row>` in this case is just a wrapper for a bunch of `<Button>` elements at the top of the pane. If it is present but empty it doesn't really affect the layout. If we had, e.g. an empty dropdown menu, then it would definitely be worth wrapping the other Row/Menu/Whatever element to suppress the display when all permission-tests return false. 

It's great that you are thinking about this kind of display issue though. Thanks for bringing it up!

Reply:
Thank you, @zburke, for clarification.